test/uk/gov/hmrc/vulnerabilities/persistence/VulnerabilitiesRepositorySpec.scala

```diff
@@ -30,9 +32,12 @@ class VulnerabilitiesRepositorySpec
   extends AnyWordSpecLike
     with Matchers
     with DefaultPlayMongoRepositorySupport[VulnerabilitySummary]
-    with IntegrationPatience {
+    with IntegrationPatience
+    with GuiceOneServerPerSuite {
 
-  override lazy val repository = new VulnerabilitySummariesRepository(mongoComponent)
+  private val appConfig = app.injector.instanceOf[AppConfig]
```

Comment:
It seems lighter to just instantiate `AppConfig` rather than mixing in `GuiceOneServerPerSuite`